test/util_skadi.cc

```diff
@@ -24,6 +24,14 @@ TEST(UtilSkadi, Grade) {
   answer = (down_grade * down_weight * 3 + up_grade * up_weight) / (down_weight * 3 + up_weight);
   grade = std::get<0>(skadi::weighted_grade({0, 1, 0, -1, -2}, 1.0));
   EXPECT_NEAR(grade, answer, .00001) << "Weighted grade was not right";
+
+  // check mean elevation
+  auto elev = std::get<3>(skadi::weighted_grade({0.0f, 15.0f, -32768.0f}, 1.0));
+  EXPECT_NEAR(elev, 7.5f, 0.00001);
+
+  // check mean elevation
```

Comment:
weighted grade is not mean elevation though its like the nominal slope of the road

Reply:
meh nevermind, just the naming is unfortunate, its grade and mean elevation

Reply:
It’s only getting the elevation value from the tuple though;)